src/utils/setup/on-create-page.js

```diff
@@ -22,6 +23,8 @@ const DEFAULT_FEATURED_LEARN_SLUGS = [
     'how-to/polymorphic-pattern',
 ];
 
+const STAGING_PAGES = ['/academia/', '/media/'];
```

Comment:
I think the discoverability of this array could be an issue. Maybe we need a place in the README for 'developing'?
 
As we add more of these arrays and values it might be beneficial to have a config file we can import and update outside of the code itself.

Reply:
I was thinking about some external source to allow these changes to not be full-on code changes. Perhaps this is a nice place for Stitch to come in?

Reply:
I was thinking about that. On the one hand it would enable stakeholders to 'un-stage', but on the other it's an impediment to contributing.

Probably still the right thing to do though

Reply:
Yeah I think there are some cons, such as adding another Stitch call to build time. I think the best course of action is to merge this PR as-is to allow us to develop on the academia page and then experiment with Stitch separately.